libcnb-cargo/tests/test.rs

```diff
@@ -330,5 +329,5 @@ fn copy_dir_recursively(source: &Path, destination: &Path) -> std::io::Result<()
 }
 
 const X86_64_UNKNOWN_LINUX_MUSL: &str = "x86_64-unknown-linux-musl";
-const TARGET_DIR_NAME: &str = "target";
 const CARGO_LIBCNB_BINARY_UNDER_TEST: &str = env!("CARGO_BIN_EXE_cargo-libcnb");
+const DEFAULT_PACKAGE_DIR_NAME: &str = "libcnb-packaged";
```

Comment:
So one downside to the default name of `libcnb-packaged` is that when `cargo libcnb package` is run in the libcnb repo (eg when testing), one ends up with an output directory that's right next to the very similarly named directory for `libcnb-package`:

<img width="179" alt="Screenshot of repo directories" src="https://github.com/heroku/libcnb.rs/assets/501702/96b7e9c5-5a80-4de7-9e95-99c441e15752">

In addition, I think the name `libcnb-packaged` is also possibly a bit redundant, given that in general we'll only ever have repos that are libcnb.rs-only (once the temporary transition from bash to libcnb.rs for our CNBs is over), so the directory name perhaps doesn't need to include the words "libcnb"?

Some alternatives that spring to mind:

```
build/
dist/
output/
packaged/
```

I'd probably lean away from `build/` since:
(a) it would never tab complete cleanly due to buildpack repo roots sometimes containing the file `buildpack.toml` (depending on repo layout),
(b) conceptually the name overlaps too much with compiling etc, so it might not be clear how it differs from the output of the compiler etc.

(Whilst the same issue about tab completion could be said for `packaged/` and `package.toml`, I think we should just delete the `package.toml` of file in the repo roots that have them, since they are currently rewritten to be correct during packging anyway.)

As such, I think my favourites would be `dist/` or `packaged/`.

Reply:
I have zero passion about the name of the directory. Happy to change. :) I changed it to `packaged` in [2919416](https://github.com/heroku/libcnb.rs/pull/583/commits/29194163b2c0239f611ca8fc89a85777bdb1d2c1).